libraries/rush-lib/src/schemas/rush.schema.json

```diff
@@ -181,6 +181,10 @@
           "description": "The commit message to use when committing change log files \"rush version\". Defaults to \"Update changelogs [skip ci]\"",
           "type": "string"
         },
+        "changefilesCommitMessage": {
```

Comment:
Add this property to the `rush init` template (here https://github.com/microsoft/rushstack/blob/main/libraries/rush-lib/assets/rush-init/rush.json)

Reply:
Done, didn't know about this, thanks!